### DISCUSSION TITLE
Multiple instances of Livewire and alpine after submiting a form and redirecting with navigate

### DISCUSSION THREAD

**Discussion by Golveira** [livewire/livewire#8046] [1. Help]
I have a modal with a form inside a livewire component, and I'm getting an error after submiting the form. The form calls a save method that redirects to another page using navigate.

```
<x-modal>
 <form wire:submit="save">
  ...
  <button type="submit">Save</button>
 </form>
</x-modal>
```

```
 public function save(): void
    {
        ...
        $this->redirect(route('products.show', $product), navigate: true);
    }
```
It works normally until I refresh page. if I try to submit the form again, these errors are shown in the console and the form stops working.
![Captura de tela 2024-02-29 163216](https://github.com/livewire/livewire/assets/30783517/6b61fa8b-c9f0-41a1-8aa4-56b02e7b8241)

I've tried to manually inject livewire assets but had no success.

**Comment by danie-ramdhani** [comment]
livewire v3 is bundled with alpine js. maybe you have Alpine js running in your `app.js` or layout?

more info: https://livewire.laravel.com/docs/installation

**Reply by Golveira** [reply]
I removed alpine from app.js. Livewire is the only when injecting it in the app. The problem seems related with changing pages with navigate.

**Reply by lrljoe** [reply]
Please share both your layout file, and your app.js, along with your config/livewire.php file 


Otherwise everyone is just guessing!

It'll almost certainly be double inclusion of Livewire/Alpine

**Comment by joshhanley** [comment]
We're in the process of investigating all wire:navigate issues and we can't replicate this with the infromation give so I'm going to close this. If you're still experiencing problems, please open a new discussion and provide code as requested (layouts, components, config). Thanks!
